graph.py

```diff
@@ -74,7 +77,8 @@ def neighbors(self, n):
         Return the edges for a node in a list
         """
         try:
-            return self.g[n]
+            edges = [node[0] for node in self.g[n]]
+            return edges
```

Comment:
this symbol is poorly named.  Better name it 'neighbors' than edges.  It doesn't contain pairs, just single values